src/components/Footer/Footer.js

```diff
@@ -79,13 +79,14 @@ const Footer = ({
 
   const voteStyle = () => {
     if (totalVote > 0) {
-      return COLORS.anon_primary;
+      return COLORS.upvote;
     }
     if (totalVote < 0) {
-      return COLORS.redalert;
+      return COLORS.downvote;
     }
-    return COLORS.balance_gray;
+    return COLORS.gray400;
   };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color codes have been changed for different vote counts. Ensure that these new colors (`COLORS.upvote`, `COLORS.downvote`, `COLORS.gray400`) are defined and provide the desired user experience.
```diff
-      return COLORS.anon_primary;
+      return COLORS.upvote;

-      return COLORS.redalert;
+      return COLORS.downvote;

-    return COLORS.balance_gray;
+    return COLORS.gray400;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->